pcdsdevices/mpod_apalis.py

```diff
@@ -42,6 +42,15 @@ class MPODApalisChannel(BaseInterface, Device):
                 kind='normal', string=True,
                 doc='MPOD Channel State [Off/On]')
 
+    desc = Cpt(EpicsSignal, ':VoltageMeasure.DESC', kind='normal',
+               doc='MPOD Channel Description')
+
+    voltage_setpoint = Cpt(EpicsSignalRO, ':VoltageSet', kind='normal',
```

Comment:
Since this is the last written setpoint, maybe we could rename this to `last_voltage_setpoint` or something to indicate that it's not the actual control value.

Reply:
I'll check tmo's scripts, and if they don't use this by name I will rename it.
If they do use it by name, I will update the description field instead.

Reply:
I'm going to rename this as suggested but include a backcompatibility alias for tmo's scripts to keep working
```
tmo/tmo_mpods.py:                mpod_dict["Voltage Set (V):"].append(mpod.voltage_setpoint.get())
tmo/tmo_cvmi_utility.py:                self.voltage_settings[key] = mpod.voltage_setpoint.get()
tmo/tmo_cvmi_utility.py:            while ( tmo_mpod.MCP_Back.voltage.get() < (0.9 * tmo_mpod.MCP_Back.voltage_setpoint.get() ) ):
tmo/tmo_cvmi_utility.py:            while ( tmo_mpod.Phosphor.voltage.get() < (0.9 * tmo_mpod.Phosphor.voltage_setpoint.get() ) ):
```